<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0" columnstretch="0,0">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1500">
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="4" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit">
          <property name="placeholderText">
           <string>Plot 2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="placeholderText">
           <string>Plot 1</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QRadioButton" name="radioButton_5">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bg_plots</string>
          </attribute>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="placeholderText">
           <string>Plot 4</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="placeholderText">
           <string>Plot 3</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string/>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bg_plots</string>
          </attribute>
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="placeholderText">
           <string>Plot 5</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string/>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bg_plots</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bg_plots</string>
          </attribute>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string/>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bg_plots</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pb_clrPlot_1">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bg_clrPlots</string>
          </attribute>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pb_clrPlot_2">
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bg_clrPlots</string>
          </attribute>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pb_clrPlot_3">
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bg_clrPlots</string>
          </attribute>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QPushButton" name="pb_clrPlot_4">
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bg_clrPlots</string>
          </attribute>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QPushButton" name="pb_clrPlot_5">
          <property name="text">
           <string/>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">bg_clrPlots</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vl_plot"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0" columnstretch="50,50,50,50">
      <item row="3" column="2">
       <widget class="QPushButton" name="pb_setPos1">
        <property name="text">
         <string>SetPos 1</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_send</string>
        </attribute>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="pb_send">
        <property name="text">
         <string>SEND</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_send</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enable:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="le_setPos3">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="cb_mode">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Set Pos:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mode:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>CONTROL</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="le_setPos">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="cb_enable">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Degrees:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pb_setPos3">
        <property name="text">
         <string>SetPos 3</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_send</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="le_setPos1">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="le_setPos2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="l_pps">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pb_setPos2">
        <property name="text">
         <string>SetPos 2</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_send</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Angle:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="cb_degrees">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lr_angle">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pb_gen">
        <property name="text">
         <string>GEN</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sb_genFreq">
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> Hz</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="sb_genAmpl">
        <property name="suffix">
         <string> Â°</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="50,50,50,50,50">
      <item row="5" column="1" colspan="2">
       <widget class="QLineEdit" name="lr_zeroShift">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="pb_saveConf">
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PWM</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QLineEdit" name="le_zeroShift">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="pb_updateConf">
        <property name="text">
         <string>UPDATE</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2">
       <widget class="QLineEdit" name="le_paD">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Read</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="pb_readConf">
        <property name="text">
         <string>READ</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PA P</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="lr_paD">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>CONFIG</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QLineEdit" name="le_paP">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="lr_pwm">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="pb_readMem">
        <property name="text">
         <string>READ mem</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QLineEdit" name="le_pwm">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="lr_paP">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PA D</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Zero Shift</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLineEdit" name="lr_speed">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="3" colspan="2">
       <widget class="QLineEdit" name="le_speed">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionClear"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaylistShuffle"/>
   </property>
   <property name="text">
    <string>Conncect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
   <property name="toolTip">
    <string>Clear plot </string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
   <property name="toolTip">
    <string>Pause plotting</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="toolTip">
    <string>Disable of drivers</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_send"/>
  <buttongroup name="bg_plots">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="bg_clrPlots">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
